cucumber/kubernetes/features/login_errors.feature

```diff
@@ -18,6 +18,10 @@ Feature: Errors emitted by the login method.
     When I login to authn-k8s as "@namespace@/pod/inventory-unauthorized" with prefix "host/conjur/authn-k8s/minikube/apps"
```

Comment:
why do we have 2 `@` here? 
Ex: `@namespace@/`

Reply:
this is a placeholder that is used for text replacements - https://github.com/cyberark/conjur/blob/host-annotations/cucumber/kubernetes/features/support/world.rb#L24